clients/admin-ui/src/features/config-wizard/AddSystemForm.tsx

```diff
@@ -12,118 +12,122 @@ import {
-      <Accordion allowToggle border="transparent">
-        <AccordionItem>
-          {({ isExpanded }) => (
-            <>
-              <h2>
-                The building blocks of your data map are the list of systems
-                that exist in your organization. Think of systems as anything
-                that might store or process data in your organization, from a
-                web application, to a database or data warehouse.
-                <AccordionButton
-                  display="inline"
-                  padding="0px"
-                  ml="5px"
-                  width="auto"
-                  color="complimentary.500"
-                >
-                  {isExpanded ? `(show less)` : `(show more)`}
-                </AccordionButton>
-              </h2>
-              <AccordionPanel padding="0px" mt="20px">
-                Let’s get started by adding systems that contain data in our
-                organization. You can speed this up by using the automated
-                scanners or adding your systems manually.
-              </AccordionPanel>
-            </>
-          )}
-        </AccordionItem>
-      </Accordion>
-      <Stack>
-        <FormControl>
-          <Stack direction="row" display="flex" alignItems="center" mb={5}>
-            <IconButton
-              aria-label="AWS"
-              boxSize={iconButtonSize}
-              minW={iconButtonSize}
-              boxShadow="base"
-              variant="ghost"
-              icon={<AWSLogoIcon boxSize="full" />}
-              onClick={() => handleChangeStep(2)}
-            />
-            <Text>Infrastructure Scan (AWS)</Text>
-            <Tooltip
-              fontSize="md"
-              label="Infrastructure scanning allows you to connect to your cloud infrastructure and automatically identify systems that should be on your data map."
-              placement="right"
-            >
-              <QuestionIcon boxSize={5} color="gray.400" />
-            </Tooltip>
-          </Stack>
-          <Stack direction="row" display="flex" alignItems="center" mb={5}>
-            <IconButton
-              aria-label="Okta"
-              boxSize={iconButtonSize}
-              minW={iconButtonSize}
-              boxShadow="base"
-              variant="ghost"
-              icon={<OktaLogoIcon boxSize="full" />}
-              onClick={() => handleChangeStep(2)}
-            />
-            <Text>System Scan (Okta)</Text>
-            <Tooltip
-              fontSize="md"
-              label="System scanning allows you to connect to your sign-on platform and automatically identify systems that should be on your data map."
-              placement="right"
-            >
-              <QuestionIcon boxSize={5} color="gray.400" />
-            </Tooltip>
-          </Stack>
-          <Stack direction="row" display="flex" alignItems="center">
-            <HStack>
+const AddSystemForm = () => {
+  const dispatch = useAppDispatch();
```

Comment:
Yeah pretty much: https://redux-toolkit.js.org/tutorials/typescript#define-typed-hooks
It provides typechecking for the actions that are actually defined in the store. (So if you `useAppDispatch({ some: "nonsense" })` it will stop you.)